client/app/dedicatedCloud/resource/upgrade/state.js

```diff
@@ -0,0 +1,15 @@
+import { controllerName } from './controller';
```

Comment:
You could've just imported `controller` :)

Reply:
Yes ! I didn't realize it was a standard view declaration !